core/src/main/java/org/eclipse/hono/util/RegistrationConstants.java

```diff
@@ -72,7 +72,7 @@
     /**
      * The name of the Device Registration API endpoint.
      */
-    public static final String REGISTRATION_ENDPOINT = "registration";
+    public static final String REGISTRATION_ENDPOINT = "devices";
```

Comment:
The value of this constant is defined by the Device Registration API. I do not think that we agreed to change that API, did we? Exposing the HTTP API on base URI `devices/` is another thing ...

Reply:
Ah, some comments on the PR mentioned how the tenant > devices > credentials hierarchy made more sense so I thought it was a general API change.

Reply:
@jbtrystram please revert this change

Reply:
done